tests/test_fetch_drugsfda.py

```diff
@@ -32,6 +32,6 @@ def test_get_nda_results(fixtures_dir: Path):
             text=json_response.read(),
         )
 
-        results = get_nda_results("207145", True)
+        results = get_nda_results("207145")
         assert results
         assert len(results) > 0
```

Comment:
what were these True values for in the test?

Reply:
Oh yeah I forgot I did this. Updated the description